src/ngraph/op/binary_convolution.cpp

```diff
@@ -114,18 +114,7 @@ void op::v1::BinaryConvolution::validate_and_infer_types()
         }
     }
 
-    element::Type result_et;
     PartialShape result_shape;
-
```

Comment:
Shouldn't the filters type be of compatible types ? Do need checks on `filters_et` here ?

Reply:
This check was wrong because data type and filters type can not be the same.